app/code/Magento/Rule/view/adminhtml/web/rules.js

```diff
@@ -220,6 +221,8 @@ define([
 
             var elem = Element.down(elemContainer, 'input.input-text');
 
+            jQuery(elem).trigger('contentUpdated');
```

Comment:
Yeah, exactly 👍 

Looks like calendar dependency is not needed anymore, isn't it? Please squash changes into a single commit also.

Reply:
Indeed. I've removed it. Thank you!

Reply:
Parameter too, please: `+], function (jQuery, calendar) {`

 If you need any assistance with squashing, just tell me.